src/base-commands/base-command.js

```diff
@@ -170,6 +170,22 @@ class BaseCommand extends Command {
   async install(name) {
     return requireInstall(name, this);
   }
+
+  get pluginConfig() {
+    if (!this._pluginConfig) {
+      const pluginName = getCommandPlugin(this);
+      this._pluginConfig = new PluginConfig(this.config.configDir, pluginName);
+    }
+    return this._pluginConfig.getConfig();
+  }
+
+  set pluginConfig(config) {
+    if (!this._pluginConfig) {
+      const pluginName = getCommandPlugin(this);
+      this._pluginConfig = new PluginConfig(this.config.configDir, pluginName);
+    }
```

Comment:
nit: this block is duplicated (only a nit since it doesn't violate the rule of 3)